common/test/acceptance/tests/studio/test_studio_accessibility_form.py

```diff
@@ -0,0 +1,76 @@
+"""
+Bok-choy tests for the Studio Accessibility Feedback page.
+"""
+from common.test.acceptance.pages.studio.index import AccessibilityPage
+from common.test.acceptance.tests.helpers import AcceptanceTest
+
+
+class AccessibilityPageTest(AcceptanceTest):
+    """
+    Test that a user can access the page and submit studio accessibility feedback.
+    """
+
+    def setUp(self):
+        """
+        Load the helper for the accessibility page.
+        """
+        super(AccessibilityPageTest, self).setUp()
+        self.accessibility_page = AccessibilityPage(self.browser)
+
+    def test_page_loads(self):
+        """
+        Test if the page loads and that there is a header and input elements.
+        """
+        self.accessibility_page.visit()
+        self.assertTrue(self.accessibility_page.header_text_on_page())
+
+    def test_successful_submit(self):
+        """
+        Test filling out the accessibility feedback form out and submitting.
+        """
+        self.accessibility_page.visit()
+        self.accessibility_page.fill_form(email='bokchoy@edx.org', name='Bok-choy', message='I\'m testing you.')
+        self.accessibility_page.submit_form()
+        self.accessibility_page.success_alert_shown()
+
+    def test_error_submit_no_email(self):
```

Comment:
Any chance we could ddt these 3 tests to avoid repetition? Don't spend too much time on it if it's not simple

Reply:
Good point, done! I also broke out the repeated code in `test_error_messages` into a helper function.